src/charts/utilization/utilization-chart.html

```diff
@@ -9,7 +9,7 @@ <h1 class="available-count pull-left">
                 <span>{{currentText}}</span>
             </div>
             <div>
-                <span> of {{chartData.total}} {{config.units}}</span>
+                <span>{{'pfUtilizationChart.percentOf' | i18next:{units: config.units, value: chartData.total} }}</span>
```

Comment:
I didn't see a directive that does value replacement. Only options but maybe I missed something.